src/sentry/stacktraces/processing.py

```diff
@@ -257,17 +268,14 @@ def normalize_stacktraces_for_grouping(data: Any, grouping_config: Any = None) -
     """
 
     stacktrace_frames = []
-    stacktrace_exceptions = []
+    stacktrace_container = []
```

Comment:
A more consistent naming.

Eventually I would like to iterate over a list of StacktraceInfo. No need for two data structures.

Reply:
```suggestion
    stacktrace_containers = []
```